pkg/proc/native/proc.go

```diff
@@ -351,3 +378,55 @@ func openRedirects(redirects [3]string, foreground bool) (stdin, stdout, stderr
 
 	return stdin, stdout, stderr, closefn, nil
 }
+
+func numValid(cctx *proc.ContinueOnceContext) int {
```

Comment:
This similarly feels like it should be a method. Anything where we're "asking" something of the specific type I think makes more sense as a method.

Reply:
Done.